other/globalcss.html

```diff
@@ -0,0 +1,526 @@
+<div data-rehype-pretty-code-fragment="">
```

Comment:
We could name it something else but the syntax highlighter that does the highlighting and such adds that fragment. It's then used to scope the specific styles like line highlights, etc. It could possibly be added to the `pre` tag, but I'd need to look into it!

Reply:
Actually I just remembered the real power. If we have a `title` for the codeblock, let's say a filename, or something of that sort. It uses this to place it in the correct location and also apply the right styles/themes to the code blocks (should we ever do a light/dark mode).